services/movie.py

```diff
@@ -0,0 +1,40 @@
+from db.models import Movie
+
+from django.db.models import QuerySet
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet[Movie]:
+    queryset = Movie.objects.all()
+
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids).distinct()
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids).distinct()
+
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
```

Comment:
Consider handling the `Movie.DoesNotExist` exception in `get_movie_by_id` to avoid unhandled exceptions if a movie with the specified ID is not found.